Documentation/cli-installation-scenarios.md

```diff
@@ -137,38 +205,29 @@ The features the script needs to support/have are:
-| --prefix                             	| -InstallDir                        	| .dotnet               	| Path to where to install the CLI bundle. The directory is created if it doesn't exist. On Linux/OSX this directory is created in the user home directory (`$HOME`). On Windows, this directory is created in `%LocalAppData%`.                 	|
-| --debug                              	| -Debug                             	| false                 	| Whether to use the "fat" packages that contain debugging symbols or not.                                                                                                                                                                       	|
-| --no-path                            	| -NoPath                            	| false                 	| Export the prefix/installdir to the path for the current session. This makes CLI tools available immidiately after install.                                                                                                                    	|
-#### Install the latest nightly CLI
+| dotnet-install.sh param (Linux, OSX) 	| dotnet-install.ps1 param (Windows) 	| Defaults              	| Description                                                                                                                                                                                                                                   	|
+|--------------------------------------	|------------------------------------	|-----------------------	|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|
+| --channel                            	| -Channel                           	| "Production"          	| Which channel (i.e. "Future", "preview", "production") to install from.                                                                                                                                                                       	|
+| --version                            	| -Version                           	| global.json or Latest 	| Which version of CLI to install; you need to specify the version as 3-part version (i.e. 1.0.0-13232). If omitted, it will default to the first global.json that contains the sdkVersion property; if that is not present it will use Latest. 	|
+| --prefix                             	| -InstallDir                        	| .dotnet               	| Path to where to install the CLI bundle. The directory is created if it doesn't exist. On Linux/OSX this directory is created in the user home directory (`$HOME`). On Windows, this directory is created in `%LocalAppData%`.                	|
```

Comment:
Supporting power shell is for consistency when people are switching, supporting also -- is for consistency with people using linux and --everything-is-like-that. I talked about that with @anurse and @glennc and we thought this is gonna be least annoying solution for both Windows and Linux users.